pkg/clusteragent/admission/mutate/autoinstrumentation/target_mutator.go

```diff
@@ -353,6 +429,31 @@ func (t targetInternal) matchesPodSelector(podLabels map[string]string) bool {
 	return t.podSelector.Matches(labels.Set(podLabels))
 }
 
+// createDefaultTarget is used when there are no targets. If a user configures enabledNamespaces and libVersions, which
+// are mutually exclusive with a list of targets, then we need to translate those configuration options into a target.
+// Additionally, if there are no targets and enabledNamespaces/libVersions are not set, the expected behavior is that
+// we would inject all SDKs to all pods. This target encompasses both of those cases.
+func createDefaultTarget(namespaces []string, pinnedLibVersions map[string]string) Target {
+	// Create a default target.
+	target := Target{
```

Comment:
Right now we add an annotation to the pod that says which target this is coming from and what the configuration is. 

Is that consistent for the behavior from namespaces? We'll have an `applied-target` with this JSON format?

What happens for the annotation? I'm not sure we should be adding an `applied-target` annotation there

Reply:
For `enabledNamespaces`, the generated target applies. I updated this change to no longer apply an annotation for local lib injection.

Reply:
Thanks!